### DISCUSSION TITLE
Executable DMG file

### DISCUSSION THREAD

**Discussion by StormBringer228** [Andrewthe13th/Inventory_Kamera#414] [Ideas or Feature requests]
As niche as MacOS gamers may be, it is currently possible to side load the Genshin Impact app to via the Playcover application in order to run Genshin Impact on MacOS. A mac specific .dmg file for running on MacOS would increase the number of people that could get their inventories scanned

**Comment by Dyl-Aeon** [comment]
I've got empathy for you MacFolk, but Hoyo can't even be bothered to write their game for macOS.
Try Wine.
[https://www.winehq.org/](https://www.winehq.org/)
If it doesn't work at first, try installing the C++ redistributables for Wine as well (You may not need to)
[https://appdb.winehq.org/objectManager.php?sClass=application&iId=5766](https://appdb.winehq.org/objectManager.php?sClass=application&iId=5766)
